.travis.yml

```diff
@@ -4,11 +4,7 @@ matrix:
   fast_finish: true
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
+  - 7.1
```

Comment:
I tried adding PHP version 5.6 but this one giving error because of a doctrine dependency, could help solve?